### PR TITLE
Sample PR

### PR DESCRIPTION
**Pull request by SSIT-user1**

### CODE REVIEW

**Review by SSIT-reviewer1 — CHANGES_REQUESTED** [review]
some test2

File: mod1/src/file-6.xoo [hunk 1]
```diff
@@ -0,0 +1,7 @@
+// dummy line 1
+// dummy line 2
+// dummy line 3
+// dummy line 4
+// dummy line 5
+// dummy line 6
+// violate HasTag rule: xoo on line 7
```
Comment (SSIT-reviewer1) [inline mod1/src/file-6.xoo:7]: Code Smell Code Smell: Has Tag ([xoo:HasTag](http://127.0.0.1:15744/organizations/default-organization/rules#rule_key=xoo%3AHasTag))

[See it in SonarQube](http://127.0.0.1:15744/project/issues?id=sample&branch=feature6&issues=AWEnWr_YlxVAawkswJ2U&open=AWEnWr_YlxVAawkswJ2U)

[ISSUE_KEY]: AWEnWr_YlxVAawkswJ2U

File: mod1/mod11/src/file-7.xoo [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+// dummy line 2
+// dummy line 3
+// dummy line 4
+// dummy line 5
+// dummy line 6
+// dummy line 7
+// dummy line 8
+// dummy line 9
+// dummy line 10
+// violate HasTag rule: xoo on line 11
```
Comment (SSIT-reviewer1) [inline mod1/mod11/src/file-7.xoo:11]: Code Smell Code Smell: Has Tag ([xoo:HasTag](http://127.0.0.1:15744/organizations/default-organization/rules#rule_key=xoo%3AHasTag))

[See it in SonarQube](http://127.0.0.1:15744/project/issues?id=sample&branch=feature6&issues=AWEnWr_PlxVAawkswJ2T&open=AWEnWr_PlxVAawkswJ2T)

[ISSUE_KEY]: AWEnWr_PlxVAawkswJ2T

File: src/file-5.xoo [hunk 3]
```diff
@@ -0,0 +1,5 @@
+// dummy line 1
+// dummy line 2
+// dummy line 3
+// dummy line 4
+// violate HasTag rule: xoo on line 5
```
Comment (SSIT-reviewer1) [inline src/file-5.xoo:5]: Code Smell Code Smell: Has Tag ([xoo:HasTag](http://127.0.0.1:15744/organizations/default-organization/rules#rule_key=xoo%3AHasTag))

[See it in SonarQube](http://127.0.0.1:15744/project/issues?id=sample&branch=feature6&issues=AWEnWr_alxVAawkswJ2V&open=AWEnWr_alxVAawkswJ2V)

[ISSUE_KEY]: AWEnWr_alxVAawkswJ2V

File: mod1/mod11/src/file-7.xoo (hunk 2 above)
Comment (SSIT-reviewer1) [reply mod1/mod11/src/file-7.xoo:11]: This issue has been resolved by a user in SonarQube

[ISSUE_STATUS]: RESOLVED

**Review by ghost — COMMENTED** [review]
test comment

for commit: 03ceabba7d97c8a095cb41f5a5352d5be24d0e4e

~Code Smell Code Smell: Has Tag ([xoo:HasTag](http://127.0.0.1:21847/coding_rules#rule_key=xoo%3AHasTag))~

[See it in SonarQube](http://127.0.0.1:21847/project/issues?id=sample&branch=feature4&issues=AWDAXlrIq9jz4mafnNS2&open=AWDAXlrIq9jz4mafnNS2)

[ISSUE_KEY]: AWDAXlrIq9jz4mafnNS2

**Pull request closed without merging**